src/Swashbuckle.NodaTime/SwaggerDocsConfigExtensions.fs

```diff
@@ -32,6 +32,16 @@ module SwaggerDocsConfigExtensions =
         config.MapType<Duration>(fun () -> schemas.Duration)
         config.MapType<DateTimeZone>(fun () -> schemas.DateTimeZone)
 
+        config.MapType<System.Nullable<Instant>>(fun () -> schemas.Instant)
+        config.MapType<System.Nullable<LocalDate>>(fun () -> schemas.LocalDate)
+        config.MapType<System.Nullable<LocalTime>>(fun () -> schemas.LocalTime)
+        config.MapType<System.Nullable<LocalDateTime>>(fun () -> schemas.LocalDateTime)
+        config.MapType<System.Nullable<OffsetDateTime>>(fun () -> schemas.OffsetDateTime)
+        config.MapType<System.Nullable<ZonedDateTime>>(fun () -> schemas.ZonedDateTime)
+        config.MapType<System.Nullable<Interval>>(fun () -> schemas.Interval)
+        config.MapType<System.Nullable<Offset>>(fun () -> schemas.Offset)
+        config.MapType<System.Nullable<Duration>>(fun () -> schemas.Duration)
```

Comment:
I noticed that nullable `Period` and `DateTimeZone` aren't configured. Did you miss those two or there is a reason for that?